app/mailers/sac_cas/application_mailer.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas.
+
+module SacCas::ApplicationMailer
+  private
+
+  def compose(recipients, content_key, locales = [])
+    return if recipients.blank?
+
+    values = values_for_placeholders(content_key)
+    custom_content_mail(recipients, content_key, values, {}, locales)
+  end
+
+  def custom_content_mail(recipients, content_key, values, headers = {}, locales = [])
+    original_locale = I18n.locale
+    locales = [I18n.locale] if locales.empty?
+    headers[:to] = use_mailing_emails(recipients)
+
+    contents = locales.map do |locale|
```

Comment:
Maybe you need to use `I18n.with_locale` here So it is set for the whole execution?